src/main/java/org/lukhnos/nnio/file/Files.java

```diff
@@ -337,14 +337,21 @@ public static long size(Path path) throws IOException {
   }
 
   public static Path walkFileTree(Path start, FileVisitor<? super Path> visitor) throws IOException {
+    File file = start.toFile();
+    if(!file.canRead()){
```

Comment:
nit: please follow the style in the file and add one space between `if` and its predicate, so `if (...)`.